ocaml-lsp-server/src/document_store.mli

```diff
@@ -8,6 +8,8 @@ val get : t -> Lsp.Uri.t -> (Document.t, Lsp.Jsonrpc.Response.Error.t) result
 
 val get_opt : t -> Lsp.Uri.t -> Document.t option
 
+val get_full_opt : t -> Lsp.Uri.t -> (Document.t * float) option
```

Comment:
I think we might as well make `value` public. The `float` argument is hard to understand otherwise.